_docs/api/content-tracing.md

```diff
@@ -36,7 +36,7 @@ redirect_from:
     - /docs/latest/api/content-tracing/
 source_url: 'https://github.com/electron/electron/blob/master/docs/api/content-tracing.md'
 excerpt: "Collect tracing data from Chromium&apos;s content module for finding performance
-bottlenecks and slow operations."
+    bottlenecks and slow operations."
```

Comment:
This was required so that `frontmatter.loadFront(apiDoc)` would load correctly, it was failing with a parse error if this was outdented fully.